front_end/panels/elements/components/DataBindingProperty.ts

```diff
@@ -291,10 +302,11 @@ export class DataBindAttributeTreeElement extends DataBindBaseTreeElement {
 
       for (let i = 0; i < evalNodes.length; i++) {
         const { evaluatableExpression, evaluatedValue, evaluationError, syncStatus, valueType } = evalNodes[i];
-        if (!!evaluationError || !syncStatus) nodeHasProblem = true;
+        if (!!evaluationError) nodeHasError = true;
```

Comment:
```suggestion
        if (evaluationError) nodeHasError = true;
```
Isn't thius the same in this case?